examples/container-deny-added-caps/template.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: templates.gatekeeper.sh/v1beta1
+apiVersion: templates.gatekeeper.sh/v1
```

Comment:
Here and throughout the examples: We're still defaulting to v1beta1, so it doesn't make sense to change this to v1.

Reply:
fixed.